### PR TITLE
fix links in gh-pages

### PR DESCRIPTION
**Pull request by roryrjb**
Just a simple fix for a couple of links only present in `gh-pages` branch that return 404s.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,11 +158,11 @@ <h2 class="project-tagline">The Homebrew package manager for Linux</h2>
...
 
-<p>Please see our <a href="https://github.com/Linuxbrew/brew/blob/master/.github/CONTRIBUTING.md#contributing-to-linuxbrew">guidelines</a> on whether to send pull requests to Linuxbrew or Homebrew.</p>
+<p>Please see our <a href="https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md#contributing-to-linuxbrew">guidelines</a> on whether to send pull requests to Linuxbrew or Homebrew.</p>
 
 <p>A good starting point for contributing is running <code>brew audit</code> (or <code>brew audit --strict</code>) with some of the packages you use (e.g. <code>brew audit wget</code> if you use <code>wget</code>) and then read through the warnings, try to fix them until <code>brew audit</code> shows no results and <a href="https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md">submit a pull request</a>. If no formulae you use have warnings you can run <code>brew audit</code> without arguments to have it run on all packages and pick one. Good luck!</p>
```
Comment (rwhogg) [inline index.html]: While you're at it, can you also change https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md to https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md ?

**Comment by bw-yuhogg** [comment]
Just clarifying here @sjackman: we use the linuxbrew.github.io repo for the website now, right?

Please excuse my confusion earlier @roryrjb and thanks a bunch for fixing the 404's!

**Comment by sjackman** [comment]
The web page is http://linuxbrew.sh
It's generated from README.md using the GitHub Automatic Page Generator at
https://github.com/Linuxbrew/brew/generated_pages/new

I'd like to migrate to Jekyll possibly using the [Beautiful Jekyll](http://deanattali.com/beautiful-jekyll/) theme. I don't have time for that right now, maybe in the new year.

**Comment by sjackman** [comment]
I've regenerated the page. I don't see the fixed page up yet at linuxbrew.sh but it should migrate soon. Ping this issue if it doesn't.

**Comment by sjackman** [comment]
Thanks for the bug report and fix, Rory!

**Comment by sjackman** [comment]
@rwhogg Ah, I understand your question now. I had forgotten about the weirdness involved with the web page repo. The web page is regenerated from the `Linuxbrew/brew` repo using:
https://github.com/Linuxbrew/brew/generated_pages/new
then select `Load README.md` and delete line 3 which reads `# Linuxbrew`
and then `Continue to Layouts`
and then `Publish page`

but to make the changes live, you have to then run

``` sh
git remote add linuxbrew.sh https://github.com/Linuxbrew/linuxbrew.github.io
git fetch origin
git push linuxbrew.sh origin/gh-pages:master
```

**Pull request closed without merging**